src/test/kotlin/it/pagopa/wallet/client/NpgClientTest.kt

```diff
@@ -172,7 +192,9 @@ class NpgClientTest {
                     500,
                     "statusText",
                     HttpHeaders.EMPTY,
-                    ByteArray(0),
+                    objectMapper.writeValueAsBytes(
```

Comment:
add verification against `spanBuilder.setAttribute` method being invoked with expected NPG response and HTTP error codes for this and below test